src/locale/locales/ca/messages.po

```diff
@@ -2024,11 +2024,11 @@ msgstr "Ves al pas anterior"
 
 #: src/view/screens/NotFound.tsx:55
 msgid "Go home"
-msgstr ""
+msgstr "Ves a l'inici"
```

Comment:
Comparing with the following string, it looks like there should be an accent on `Vés`?

Reply:
thanks, "ves" is always without accent, I changed evereywhere now